config/locales/en.yml

```
@@ -645,6 +653,38 @@ en:
     optional: "%{question_text} (optional)"
     question: Question
     submit_save: Save question
+  payment_link_form:
+    body_html: |
+      <p>You can set up a payment page (called a ‘payment link’) on GOV.UK Pay - this lets someone make a payment after submitting their form. </p>
+
+      <p>You’ll first need to create your payment link in GOV.UK Pay. You can then copy and paste its URL in the box below.</p>
+    heading: Add a link to a payment page on GOV.UK Pay
+    how_this_will_work:
+      body_html: |
+        <p>Once someone's submitted their form they'll see a confirmation page showing: </p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>a blue banner saying “You still need to pay”</li>
+          <li>their GOV.UK Forms reference number</li>
+          <li>a green ‘Continue to pay’ button - this will take them to GOV.UK Pay to make their payment</li>
+        </ul>
+
+        <p>The reference number will also be included in a confirmation email,  someone chooses to receive this - along with any 'what happens next' information you've added.</p>
```

I don't understand this sentence, and it doesn't seem to match anything in the Mural board linked from the ticket?

---

The content from the Mural is currently getting updated based on UR findings, so it's going to be a bit disconnected. I can change it to match what's there currently though

---

I think it's missing an 'if' (almost certainly copy-paste error on my part), which might be why it's hard to understand:

```suggestion
        <p>The reference number will also be included in a confirmation email, if someone chooses to receive this - along with any 'what happens next' information you've added.</p>
```

---

Ah yep that's the one!